Plugin/src/SofaPython3/PythonFactory.cpp

```diff
@@ -94,8 +94,10 @@ void PythonFactory::registerType(const std::string& typeName, dataCreatorFunctio
 
 std::map<std::string, componentDowncastingFunction>::iterator PythonFactory::searchLowestCastAvailable(const sofa::core::objectmodel::BaseClass* metaclass)
 {
+    std::string class_name = metaclass->className + "<" + metaclass->templateName + ">";
```

Comment:
I think typeName (instead of className) is supposed to be used for that ?

Reply:
Indeed, I'll change it. Was it there before your PR on class reflection?

Reply:
Before.

Reply:
Sure about that?  ;-)
https://github.com/sofa-framework/sofa/pull/1283/files#diff-7018344f99b56549b25fa7e0ea6468f8R62

I'm in a deadline this week so I won't update SOFA to get your class reflection feature. I'll try to do it this weekend and update this PR consequently.